src/auth-service/utils/mailer.js

```diff
@@ -350,6 +359,15 @@ const mailer = {
         };
       }
 
+      if (email === "automated-tests@airqo.net") {
```

Comment:
@BenjaminSsempala , in each of our mailing functions, add an exception for this particular email address so that we always just return a success whenever it is encountered. 

In the near future of Q1 2024, we could also just enhance this and have it as an array of emails picked from a database so that we can future proof this part of adding exceptions for email sending.